example-overrides/local-overrides-osp16-2-with-pcipassthrough-ovs-hw-offload.yaml

```diff
@@ -0,0 +1,19 @@
+# This example will deploy OSP 16.2 with OVS Hardware offload configured
+# with SR-IOV passthrough for enp130s0f0 NIC, with hugepages and CPU partitioning.
+standalone_host: <standalone FQDN>
+public_api: <IP address used to reach the node>
+rhos_release: 16.2
+sriov_interface: enp130s0f0
+sriov_nic_numvfs: 63
+sriov_nova_pci_passthrough:
+  - devname: "enp130s0f0"
+    physical_network: "hostonly"
+kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=128 iommu=pt intel_iommu=on isolcpus=4-47"
```

Comment:
Do you want VMs to use these hugepages? If so you'll need to update the OpenStack flavors, too.

Reply:
yeah the flavors are created/updated manually for now, I guess this could be a new (separate) feature to dev-install. Although too opinionated in my taste. Because there are many ways of configuring the flavor.